src/types.ts

```diff
@@ -10,3 +12,13 @@ export enum Disease {
     Laminopathy = "Laminopathy",
     SkeletalMyopathy = "Skeletal Myopathy",
 }
+
+export interface Diagram {
```

Comment:
this seems like it should be a props interface, not a separate type out here (since it'll probably only be used where the component is)